src/workerd/api/node/zlib-util.c++

```diff
@@ -476,11 +484,11 @@ void CompressionStream<CompressionContext>::writeStream(jsg::Lock& js,
 
   writing = true;
 
-  context.setBuffers(input, inputLength, output, outputLength);
-  context.setFlush(flush);
+  context()->setBuffers(input, inputLength, output, outputLength);
+  context()->setFlush(flush);
```

Comment:
Nit: Can `context()` return the reference instead of a pointer? ...`context().setBuffer(...)` just looks cleaner

Reply:
I'll follow up with a new pull-request for this.